routes_test.go

```diff
@@ -59,6 +64,33 @@ func TestRouteOk(t *testing.T) {
 	}
 }
 
+// TestDeleteMethod tests. Test compatibility 
+// to http spec with method names
+
+func TestRouteDeleteMethod(t *testing.T) {
+	r, _ := http.NewRequest("DELETE", "/person/anderson/thomas?learn=kungfu", nil)
+	w := httptest.NewRecorder()
+
+	handler := new(RouteMux)
+	handler.Delete("/person/:last/:first", HandlerEmpty)
+	handler.ServeHTTP(w, r)
+	
+	if w.Code != http.StatusOK {
+		t.Errorf("Basic check for DELETE event failed")
+	}
+}
+
+
```

Comment:
There's some unnecessary whitespace here. (I'm not even part of this project, but you probably want to remove it.)